amanuensis/blueprints/filterset.py

```diff
@@ -95,6 +124,9 @@ def delete_search(search_id):
             "Unable to load or find the user, check your token"
         )
 
-    response = flask.jsonify(delete(logged_user_id, search_id))
+    # get the explorer_id from the querystring
+    explorer_id = flask.request.args.get('explorer', default=1, type=int)
```

Comment:
I've been going back and forth on this, but now I think it's better to rename the search param to `explorerId` to match the variable name. In my view, `explorerId=1` is simply more explicit than `explorer=1`.

Reply:
Thank you for making the change.